cmd/buildkitd/main_oci_worker.go

```diff
@@ -190,6 +198,13 @@ func ociWorkerInitializer(c *cli.Context, common workerInitializerOpt) ([]worker
 
 	cfg := common.config.Workers.OCI
 
+	if cfg.OCIBinary != "" {
+		// If the command-line flag for using a different OCI worker binary is set, buildkit-runc will be permanently changed
+		// to use said OCI worker binary until it is changed again.
+		exec.Command("/bin/sh", "-c", "sudo rm -f $(which buildkit-runc)").Run()
+		exec.Command("/bin/sh", "-c", "sudo ln -s $(which "+cfg.OCIBinary+") /usr/local/bin/buildkit-runc").Run()
+	}
```

Comment:
No, you don't need to replace the binary. Just configure the worker to use the binary.

Reply:
Is the runc worker in moby/buildkit/worker/runc just another name for OCI worker?

Reply:
Yes, they are the same thing.